ajax/libs/dragdealer/package.json

```diff
@@ -7,6 +7,7 @@
     "url": "git://github.com/skidding/dragdealer.git"
   },
   "dependencies": {},
+  "license":"MIT",
```

Comment:
@x112358 , please add a blank after : , such as "license": "MIT"
Please tag me after you fix it, thanks!